src/mcts/node.h

```diff
@@ -165,6 +165,7 @@ class Node {
   // Returns whether the node is known to be draw/lose/win.
   bool IsTerminal() const { return terminal_type_ != Terminal::NonTerminal; }
   bool IsTbTerminal() const { return terminal_type_ == Terminal::Tablebase; }
+  bool IsTwoFoldTerminal() const { return terminal_type_ == Terminal::TwoFold; }
```

Comment:
Previously I believe crem suggested not adding the helper method yet for just a single use, so just inline it instead.

Reply:
What do you mean by this?

Reply:
In this case I believe having helpers for all is better for consistency.

Reply:
I agree, consistency here is preferred. I will mark this as resolved then.